packages/utils/src/getActiveLoginIDFromLocalStorage.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Get active loginid from localStorage
+ * @returns {string} - Active loginid
+ */
+const getActiveLoginIDFromLocalStorage = (): string => {
```

Comment:
```suggestion
const getActiveLoginIDFromLocalStorage = () => {
```
@sergei-deriv Let TS infer whenever possible.